rel-eng/packages/osbs-client

```diff
@@ -0,0 +1 @@
+0.36-1 ./
```

Comment:
This is a pointer for tito of a current version and package subfolder. This file would be automatically updated on `tito release` - but I think it would work even if this version is outdated